packages/ui/src/components/button/button.native.tsx

```diff
@@ -1,8 +1,9 @@
 import { TouchableOpacity, ViewStyle } from 'react-native';
 
 import { ResponsiveValue } from '@shopify/restyle';
-import { Theme } from 'native';
 
+// FIXME issues#64: This is a temporary fix to commit as I broke the direct import of the theme
+import { Theme } from '../../theme-native';
```

Comment:
@edgarkhanzadian I'm not sure how I broke this but on refactor to a `function` to appease eslint, this import started failing so I changed it to a relative path